src/parser/shaman/elemental/modules/azerite/LavaShock.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Analyzer from 'parser/core/Analyzer';
+import { formatNumber } from 'common/format';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+
+class EchoOfTheElementals extends Analyzer {
+  procs = 0;
+  damageGained = 0;
+  relevantData=null;
```

Comment:
Not used anywhere?

Reply:
relevantData is the attribute I write the necessary summon+damage spellids to